plugins/sfModsPlugin/lib/sfModsConvertor.class.php

```diff
@@ -339,6 +339,7 @@ protected function electronicLocatorTag()
         file_put_contents($tempFile, curl_exec($curlSession));
 
         curl_close($curlSession);
```

Comment:
`curl_close()` can be removed as it no longer does anything. See PHP 8 breaking changes.